examples/web-wagmi/src/profiles/components/ProfileSelector.tsx

```diff
@@ -1,16 +1,19 @@
 import { useExploreProfiles, ProfileFragment } from '@lens-protocol/react';
 
+import { ErrorMessage } from '../../components/error/ErrorMessage';
 import { invariant } from '../../utils';
 
-type SelectProfileProps = {
+type ProfileSelectorProps = {
   onProfileSelected: (profile: ProfileFragment | null) => void;
 };
 
-export function SelectProfile({ onProfileSelected }: SelectProfileProps) {
-  const { data, loading } = useExploreProfiles({ limit: 30 });
+export function ProfileSelector({ onProfileSelected }: ProfileSelectorProps) {
```

Comment:
This is a much better name, thanks!

Reply:
oh, I didn't come up with it, I just used the file name.